packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,288 @@
+import {
+  StackLayout,
+  Avatar,
+  Text,
+  Card,
+  Button,
+  Tooltip,
+  FlexLayout,
+  FlowLayout,
+} from "@salt-ds/core";
+import {
+  MessageIcon,
+  LocationIcon,
+  CallIcon,
+  ChatIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  UserBadgeIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+import { useState } from "react";
+import persona from "../../assets/avatar.png";
+
+export default {
+  title: "Patterns/Contact Details",
+} as Meta;
+
+const basicContact = {
+  primary: "Jane Doe",
+  secondary: "This is a long secondary title to show that the text will wrap",
+  metadata: [
+    {
+      label: "Role",
+      value: "Analyst",
+      icon: <UserBadgeIcon aria-hidden />,
+    },
+    {
+      label: "Location",
+      value: "London, GBR",
+      icon: <LocationIcon aria-hidden />,
+    },
+    {
+      label: "Office",
+      value: "+44 2077 431102",
+      icon: <CallIcon aria-hidden />,
+    },
+    {
+      label: "Email",
+      value: "email@company.com",
+      icon: <MessageIcon aria-hidden />,
+    },
+  ],
+};
+
+export const ContactDetails = ({ avatarSrc = persona }) => {
+  return (
+    <StackLayout
+      direction={"row"}
+      gap={2}
+      style={{
+        width: "400px",
+      }}
+    >
+      <Avatar
+        src={avatarSrc as string}
+        aria-label={basicContact.primary}
+        fallbackIcon={<UserBadgeIcon />}
```

Comment:
Is this the correct fallback icon?

Reply:
@yunjungyeh Darren and I thought the User Badge Icon was the most appropriate, you happy with that ?